build.sbt

```diff
@@ -83,3 +83,6 @@ ScalariformKeys.preferences := ScalariformKeys.preferences.value
   .setPreference(PreserveDanglingCloseParenthesis, true)
 
 fork in run := true
+
+
+fork in run := true
```

Comment:
I am assuming these edits to the build file were unintentional? If so, please remove them! And before committing anything, always do a `git status` and make sure the files that are marked as modified are what you expect.

Reply:
Okay should be good to go now

Reply:
@misaugstad let me know if there is anything else :)